APCAppCore/APCAppCore/Library/CMS/APCCMS.m

```diff
@@ -32,20 +32,23 @@
 // 
  
 #import "APCCMS.h"
+#import "APCCMSSupport.h"
 
-/**
- Remember, only include ONE of these encryption .m files
- in your project.  See ENCRYPTION_README.txt for more
- information.
- */
 NSData * cmsEncrypt (NSData *data,
                      NSString * __unused identityPath,
                      NSError **error)
 {
+    NSData *encryptedData = data;
+    
     if (error != nil)
     {
         *error = nil;
     }
+    
+    Class support = NSClassFromString(@"APCCMSSupport");
+    if (support != Nil) {
+        encryptedData = [support cmsEncrypt:data identityPath:identityPath error:error];
+    }
```

Comment:
Would be better to throw a warning log if `APCCMSSupport` is not exist?
To alert the developer that the data is not being encrypted?